### PR TITLE
Send email confirmation for cancel/reactivate

### PR DESCRIPTION
**Pull request by stephencelis**
Knocking out a coupla these.

### CODE REVIEW

File: Sources/PointFree/Account/Account.swift [hunk 1]
```diff
@@ -182,8 +182,9 @@ private func planName(for subscription: Stripe.Subscription) -> String {
 public func status(for subscription: Stripe.Subscription) -> String {
   switch subscription.status {
   case .active:
-    let currentPeriodEndString = subscription.currentPeriodEnd
-      .map { subscription.cancelAtPeriodEnd ? " through " + dateFormatter.string(from: $0) : "" } ?? ""
+    let currentPeriodEndString = subscription.cancelAtPeriodEnd
```
Comment (stephencelis) [inline Sources/PointFree/Account/Account.swift]: Looks like `current_period_end` always has a value from Stripe.

File: Sources/PointFree/Account/Account.swift [hunk 2]
```diff
@@ -480,10 +479,11 @@ private let subscriptionPaymentInfoView = View<Stripe.Subscription> { subscripti
...
 }
 
-private func totalAmount(for subscription: Stripe.Subscription) -> String? {
+private func totalAmount(for subscription: Stripe.Subscription) -> String {
   let totalCents = subscription.plan.amount.rawValue * subscription.quantity
-  let totalDollars = Double(totalCents) / 100
-  return currencyFormatter.string(from: NSNumber(value: totalDollars))
+  let totalDollars = NSNumber(value: Double(totalCents) / 100)
+  return currencyFormatter.string(from: totalDollars)
+    ?? NumberFormatter.localizedString(from: totalDollars, number: .currency)
```
Comment (stephencelis) [inline Sources/PointFree/Account/Account.swift]: `NumberFormatter.string(from:)` returns an optional, but `localizedString(from:)` does not 😕

File: Sources/PointFree/Account/Cancel.swift [hunk 3]
```diff
@@ -73,23 +75,23 @@ private func cancel(_ conn: Conn<StatusLineOpen, Tuple2<Stripe.Subscription, Dat
...
             )
-          ),
-          const(
-            conn |> redirect(
-              to: .account(.index),
-              headersMiddleware: flash(.notice, "We’ve canceled your subscription.")
-            )
           )
-        )
+        ) { _ in
```
Comment (mbrandonw) [inline Sources/PointFree/Account/Cancel.swift]: whoa my brain is having a hard time parsing this. is this a trailing closure for a curried `either` or something?

Comment (stephencelis) [reply Sources/PointFree/Account/Cancel.swift]: Ah! It's `either(L -> A, B -> A)`. The left path is a const so I pass it through. I'm using a trailing closure for the "right" path.

**Pull request merged**